gapic_plugin.py

```diff
@@ -157,10 +157,17 @@ def main(data):
   return output
 
 if __name__ == '__main__':
+  try:
```

Comment:
This is necessary to work with binary data in Python 3 - see the note here: https://docs.python.org/3/library/sys.html#sys.stdin